### ISSUE TITLE
 ValueError: target is not a valid column of X

### ISSUE THREAD

**Issue by zemlyansky** [dabl/dabl#193]
`dabl.plot(df, 'target')`, `dabl.plot(X=df, y='target')`, `dabl.plot(X=df, target_col='target')` produce the same error:
```
~/.local/lib/python3.7/site-packages/dabl/plot/utils.py in _check_X_target_col(X, target_col, types, type_hints, task)
    319                          "got {}".format(target_col))
    320     if target_col not in X.columns:
--> 321         raise ValueError("{} is not a valid column of X".format(target_col))
    322 
    323     if X[target_col].nunique() < 2:

ValueError: target is not a valid column of X
```
Checking that condition manually:
```
'target' not in df.columns
Out: False
```

**Comment by amueller** [comment]
Thank you for reporting!
This might be #160 which is fixed in master. Is there a warning that 'target' is dropped? Which version are you using?

A work-around for older versions is ``type_hints={'target':categorical}`` (for classification, or 'continuous' for regression).

**Comment by zemlyansky** [comment]
Version: '0.1.6' installed with `pip`
Target is imbalanced, yes
`detect_type` for `target` shows:
```
continuous | dirty_float | low_card_int | categorical | date | free_string | useless
False | False | False | False | False | False | True
```
`type_hints={'target':'categorical'}` helped
Thanks!

**Comment by amueller** [comment]
Thanks for investigating! I need to make a new release (again) it seems like.

**Comment by amueller** [comment]
I just released dabl 0.1.7 which has a fix. So if you ``pip install dabl==0.1.7`` the problem should go away. Also, the class histograms should be more useful after the update, for very imbalanced data they were kind of useless before, I felt (feedback welcome).
